# 9章のメモ(例外処理)

## 例外の捕捉(Catch)

例外が起きたら、普通はそこで処理が止まる  
それ以降も継続したい場合はCatchが必要

Rubyでは下のようにかく

```ruby
begin
  # 例外が起きうる処理
rescue
  # 例外が発生した場合の処理
end
```